\begin{table*}
    \caption{Open Source Variant Monitoring Tools}
    \label{tab:0704a}
    \begin{tabularx}{\textwidth}{sb}
        \toprule
            Monitor Tool & Description \\
        \midrule
            \href{https://covidcg.org}{COVID-19 CG} &
            COVID-19 CG is a dashboard for tracking SARS-CoV-2 single-nucleotide variations (SNVs) and lineages according to location, date, gene, and mutation of interest.\cite{Chen2020} \\

            \href{https://cov-lineages.org}{CoV-Lineages} &
            Cov-lineages use a nomenclature system called PANGO Lineage. The system highlights active virus lineages in different locations to identify those that contribute to the most spread.\cite{Rambaut2020} \\

            \href{https://covariants.org}{CoVariants} &
            CoVariants shows an overview of SARS-CoV-2 mutations and variants that are rapidly spreading. The website also provides additional details such as specific mutations, possible impacts, and a table on different nomenclatures used. \\

            \href{https://outbreak.info/compare-lineages}{outbreak.info} &
            Outbreak.info gathers both SARS-CoV-2 and COVID-19 information into a user-friendly website to track emerging mutations and epidemiological trends. \\

            \href{https://nextstrain.org/blog/2021-01-06-updated-SARS-CoV-2-clade-naming}{Nextstrain} &
            Nextstrain provides constant updates on SARS-CoV-2 mutation and evolution through interactive data visualizations and bioinformatic analyses. They also use their own lineage nomenclature based on the SARS-CoV-2 cladogram. \\
        \bottomrule
    \end{tabularx}%
\end{table*}